### PR TITLE
Add db connector

### PR DESCRIPTION
**Pull request by rmweir**
Part of ENG-5007.

### CODE REVIEW

**Review by ThomasK33 — CHANGES_REQUESTED** [review]
You'll also have to add this feature to `admin-apis/pkg/licenseapi/license_new.go`

File: pkg/licenseapi/features.go [hunk 1]
```diff
@@ -85,5 +86,6 @@ func GetVClusterFeatures() []FeatureName {
 		VirtualClusterProDistroFips,
 		VirtualClusterProDistroIntegrationsKubeVirt,
 		VirtualClusterProDistroIntegrationsExternalSecrets,
+		VirtualClusterConnectorExternalDatabase,
```
Comment (FabianKramm) [inline pkg/licenseapi/features.go:89]: I would call that `VirtualClusterSharedExternalDatabase` like Denise is using when talking about that feature

File: pkg/licenseapi/license_new.go [hunk 2]
```diff
@@ -141,6 +141,11 @@ func New(product ProductName) *License {
 						Name:        string(VirtualClusterProDistroFips),
 						Status:      virtualClusterStatus,
 					},
+					{
+						DisplayName: "Database Connectors",
```
Comment (FabianKramm) [inline pkg/licenseapi/license_new.go:145]: I would call it Shared External Database

**Review by FabianKramm — CHANGES_REQUESTED** [review]
If you put the feature under pro distro it should be called in the following form: `VirtualClusterProDistroDatabaseConnector` since all pro distro features share that prefix

File: pkg/licenseapi/features.go (hunk 1 above)
Comment (deniseschannon) [reply pkg/licenseapi/features.go:89]: No, we want it to be called connector as it's technically implemented as a connector.

**Review by FabianKramm — APPROVED** [review]

**Pull request merged**